### DISCUSSION TITLE
Going back in repositories view loses place in list

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44177] [Mobile] [answered]
GitHub Mobile beta
1. Visit the Microsoft organization
2. Select repositories
3. Filter for sources
4. Scroll down a ways
5. Select a repository
    ![Screenshot_20230112-224456.png](https://user-images.githubusercontent.com/2119212/212232863-f695c81b-07c2-4fbb-bdf9-ca07c629fdf3.png)
6. Add star ⭐
7. Add to list
8. Go back
9. Expect to see the repository that you just added exactly where it was at step 5

### Actual results

You're back at the top of the list.

This gets more painful the more items you try to add using this process as they'll be further down the list and you'll have to re-scroll further each time.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref does this happen consistently? I can't reproduce this behavior.

Would you happen to have any developer settings on?

**Reply by Magdenko3777** [reply]
I'm

**Reply by jsoref** [reply]
Other than using beta, no.

Yes, it was consistent.

**Reply by stevepopovich** [reply]
It was consistent? Is it still happening?

**Reply by jsoref** [reply]
was = "I'm walking home in cold mist and don't want to try to take a screen recording to reproduce".

it's warmer inside... I'll verify and provide a screen recording...

**Reply by jsoref** [reply]
Yes, I can reproduce, but the video is ~80mb.

**Reply by jsoref** [reply]
https://user-images.githubusercontent.com/2119212/213018012-8327e849-46ec-4501-94ca-6593c70d5d9a.mov

**Reply by jsoref** [reply]
Actually, maybe it isn't top. But it clearly isn't where it was in the position before -- as the item I was looking at isn't onscreen.

**Reply by stevepopovich** [reply]
Ok, I see now. Thanks for the report. I will pass along to the team for prioritization.

**Reply by jsoref** [reply]
(I want to mark your last reply as the answer, but discussions won't let me do that!)
